ServicesDelete.php

```
@@ -0,0 +1,237 @@
+                if (!$this->promptAndValidatePassword()) {
+                    continue;
+                }
+                DeleteResourceJob::dispatch($toDelete);
+                $this->info('Database deletion dispatched.');
+            }
+        }
+    }
+
+    private function deleteService()
+    {
+        $services = Service::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Service' in method 'deleteService'. (undefined)

(StaticAccess)